manifests/cluster-manager/cluster-manager-registration-serviceaccount.yaml

```diff
@@ -3,3 +3,7 @@ kind: ServiceAccount
 metadata:
   name: {{ .ClusterManagerName }}-registration-controller-sa
   namespace: {{ .ClusterManagerNamespace }}
+{{if .ImagePullSecret}}
+imagePullSecrets:
+- name: {{ .ImagePullSecret }}
```

Comment:
I would expect missing imagepullsecret refs to be ignored, so if you avoid having a field, this manifest could just always list it.